config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
Right, but that return value lets you know whether or not the plugin was loaded with routes. If it wasn't you can load them here.